passman.py

```diff
@@ -77,7 +78,18 @@ def btn_Edit():
         renderCreateView(index=ind)
 
     def btn_CopyPassword():
-        print "copying password"
+        current_selection = entries_list.curselection() 
+        if current_selection:
+            selected_entry = entries_list.get(current_selection[0])
```

Comment:
This just pulls the text of the selected entry from the list, which is the name of the website. So this will end up copying the name of the website to the clipboard rather than the password.